easybuild/tools/github.py

```diff
@@ -324,6 +325,21 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
         # make sure path exists, create it if necessary
         mkdir(path, parents=True)
 
+    develop = download_repo(repo=GITHUB_EASYCONFIGS_REPO, branch='develop', path=path)
+    if not os.path.isdir(develop):
+        raise EasyBuildError("Downloading of develop branch failed: not found in %s", develop)
+
+    patch_name = 'patch%s' % pr
+    patch_url = URL_SEPARATOR.join([GITHUB_URL, GITHUB_EB_MAIN, GITHUB_EASYCONFIGS_REPO, GITHUB_PR, '%s.patch' % pr])
+    patch_path = os.path.join(patch_url, patch_name)
+    patch = download_file(patch_name, patch_url, patch_path)
+    if patch is None:
+        raise EasyBuildError("Failed to download file %s to %s", patch_url, patch_path)
+
+    result = apply_patch(patch, develop, level=1)
```

Comment:
you'll need to apply the patch with `-N`:

```
       -N  or  --forward
          Ignore patches that seem to be reversed or already applied.  See also -R.
```

otherwise using `--from-pr` on a PR that is already merged into `develop` will not work, cfr. the broken tests in https://travis-ci.org/hpcugent/easybuild-framework/jobs/142994798